rst2pdf/tests/input/test_753.txt

```diff
@@ -0,0 +1,27 @@
+.. image:: images/biohazard.png
```

Comment:
Can we add a page before this page in the PDF that explains what the next pages should look like as that'll make it easier for future people to confirm that the test is passing?

Reply:
Done!